miniapps/solvers/lor_solvers.cpp

```diff
@@ -0,0 +1,200 @@
+// Copyright (c) 2010-2021, Lawrence Livermore National Security, LLC. Produced
+// at the Lawrence Livermore National Laboratory. All Rights reserved. See files
+// LICENSE and NOTICE for details. LLNL-CODE-806117.
+//
+// This file is part of the MFEM library. For more information and source code
+// availability visit https://mfem.org.
+//
+// MFEM is free software; you can redistribute it and/or modify it under the
+// terms of the BSD-3 license. We welcome feedback and contributions, see file
+// CONTRIBUTING.md for details.
+//
+//                     ---------------------------------
+//                     Low-Order Refined Solvers Miniapp
+//                     ---------------------------------
+//
+// This miniapp illustrates the use of low-order refined preconditioners for
+// finite element problems defined using H1, H(curl), H(div), or L2 finite
+// element spaces. The following problems are solved, depending on the chosen
+// finite element space:
+//
+// H1 and L2: definite Helmholtz problem, u - Delta u = f
+//            (in L2 discretized using the symmetric interior penalty DG method)
+//
+// H(curl):   definite Maxwell problem, u + curl curl u = f
+//
+// H(div):    grad-div problem, u - grad(div u) = f
+//
+// In each case, the high-order finite element problem is preconditioned using a
+// low-order finite element discretization defined on a Gauss-Lobatto refined
```

Comment:
The Dorhmann paper refers to the refined mesh as GLL. Is there a reason for the different name here?

Reply:
I suppose Gauss-Lobatto, Gauss-Legendre-Lobatto (GLL), and Legendre-Gauss-Lobatto are all names for the same thing. Perhaps in abbreviations, people like to use GLL for Lobatto, so that GL can be used for Gauss-Legendre without ambiguity.

If we are spelling out the name in full, I think Gauss-Lobatto is fine. It's also consistent with MFEM's `BasisType`.